en/python_installation/instructions.md

```diff
@@ -52,10 +48,10 @@ It is very likely that you already have Python installed out of the box. To chec
 {% filename %}command-line{% endfilename %}
 ```
 $ python3 --version
-Python 3.6.1
+Python {{ book.py_release }}
 ```
 
-If you have a different version of Python installed, at least 3.4.0 (e.g. 3.6.0), then you don't have to upgrade. If you don't have Python installed, or if you want a different version, first check what Linux distribution you are using with the following command:
+If you have a different version of Python installed, at least {{ book.py_min_version }} (e.g. {{ book.py_release }}), then you don't have to upgrade. If you don't have Python installed, or if you want a different version, first check what Linux distribution you are using with the following command:
```

Comment:
`{{ book.py_release }}` can't be an example of "**different** version of Python [than `{{ book.py_release }}`]".

I think it should be
>  If you have a different version of Python installed, at least 3.6.0 (e.g. 3.6.8), then you don't have to upgrade. If you don't have Python installed, or if you want a different version, first check what Linux distribution you are using with the following command:

Maybe we should set `book.py_min_version` back to `3.6.0` and introduce another variable for the fully patched release of the same minor version. That'd also make the other statements where `{{ book.py_release }}` is now used more true, as the requirement really _is_ 3.6._0_ or higher, and not the latest security release.

Reply:
How about this:

```
"py_min_version": 3.6,
"py_min_release": 3.6.8,
```

We can use it as follow:

... at least {{ book.py_min_version }} (e.g. {{ book.py_min_release }})

Reply:
Yeah, that sounds good.

Reply:
Added https://github.com/DjangoGirls/tutorial/pull/1683/commits/1958088cceaa83483f4eae96a8a04ff60e519b79